kit/utils/constants.py

```diff
@@ -26,6 +26,22 @@ class Constants:  # pylint: disable=too-many-instance-attributes
     # Root directory
     HEKIT_ROOT_DIR: Path = Path(__file__).resolve().parent.parent.parent
 
+    # HE comands
+    HEKIT_COMMANDS = [
+        "check-dependencies",
+        "docker-build",
+        "init",
+        "install",
+        "build",
+        "fetch",
+        "list",
+        "new",
+        "plugins",
+        "remove",
+        "algebras",
+        "gen-primes",
+    ]
+
```

Comment:
Maybe a set would be better?
```suggestion
    HEKIT_COMMANDS = {
        "check-dependencies",
        "docker-build",
        "init",
        "install",
        "build",
        "fetch",
        "list",
        "new",
        "plugins",
        "remove",
        "algebras",
        "gen-primes",
    }

```

Reply:
updated